LettersCount/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace LettersCount
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the word:");
+            string word = Console.ReadLine();
+            int wordL = word.Length - 1;
```

Comment:
you can remove it and use

`for (int i = 0; i < wordL; i++)` instead of `for (int i = 0; i <= wordL; i++)`